### PR TITLE
Update Slate project status - Low maintenance

### PR DESCRIPTION
**Pull request by wizardlyhel**
Update README and git templates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,36 @@
 
 ![slate animated banner](https://user-images.githubusercontent.com/4837696/47506317-cbe22400-d83d-11e8-9867-1dc874943833.gif)
 
+## Slate on Low Maintenance
+Due to project priority and resource constraints, this project is currently on low maintenance. We recognize that there are a lot of activities around this package, however, we unable to accommodate the maintenance this project requires.
```
Comment (chrisberthe) [inline README.md]:
```suggestion
Due to project priority and resource constraints, this project is currently on low maintenance. We recognize that there are a lot of activities around this package. However, we are unable to accommodate the maintenance this project requires.
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,36 @@
...
+- New feature implementation
+- Existing feature changes
+- Major bugs due to any tools version changes
+- Major bugs due to Shopify internal changes
+- Minor bugs
+- Will not evaluate and merge any community created PRs
+- Will not address any community reported issues except ones that implicate Shopify security
+
+### Should I continue using Slate?
+You may. As it is, Slate is still useful for themes development.
```
Comment (chrisberthe) [inline README.md]:
```suggestion
You may. As it is, Slate is still useful for Shopify theme development.
```

File: README.md [hunk 3]
```diff
@@ -2,6 +2,36 @@
...
+
+### What happened? Why are you stopping? Will Shopify work on Slate in the future?
+Due to Shopify priorities and resource constraints, we have decided to put more emphasis on another project rather than Slate. We have plans to return back to Slate when our priorities shifts back to Slate. While evaluating the current state of Slate, there are two key developer workflows that we are not satisfied with:
```
Comment (chrisberthe) [inline README.md]:
```suggestion
Due to Shopify priorities and resource constraints, we have decided to put more emphasis on another projects. We have plans to return to Slate once our priorities change. While evaluating the current state of Slate, there are two key developer workflows that we are not satisfied with:
```

**Comment by dan-gamble** [comment]
Is this likely to affect https://github.com/shopify/theme-scripts as well?

**Comment by jonathanmoore** [comment]
Yikes! This is a pretty big setback, hopefully, other solutions and tools will emerge. I really liked where Slate was headed. 😞

**Comment by wizardlyhel** [comment]
@dan-gamble theme-script is not affected. It is an individual utilities library that can be used independently without Slate.

@jonathanmoore Thank you for your feedback. This is not an easy decision either. With the increasing Slate usage week by week, as of late too, it is really hard for us to make this decision as well. However, it is also why we put it in low maintenance. We want to be better equipped when we do come back to Slate, so that we could tackle on the real, hard problems.

**Comment by ruinyourfun** [comment]
Hire more talented developers as the ones you already have :) Anyway, hope you'll get back to it soon, this is a pretty ambitious & great project that I'll still use.

**Pull request merged**